spec/dummy/app/themes/default/views/layouts/thredded.html.erb

```diff
@@ -9,7 +9,16 @@
 
   <%= content_tag(:body, id: (yield :thredded_page_id), class: 'layout-application') do %>
     <%= render 'thredded/shared/icons_svg' %>
-    <%= yield %>
+
+    <div class="main-container">
+      <header class="navigation" role="banner">
+        <%#= render 'thredded/shared/top_nav' %>
+        <%#= render 'thredded/shared/topic_nav' %>
```

Comment:
No, sorry. I want to render those layouts but I don't know what the right directory is for them now since I'm rendering them in `layouts/thredded.html` and they are in `thredded/shared/*`. That make it any clearer?

Now that I'm typing this should I move those partials into `layouts`?